app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
I think `sanitize` is for HTML. One of the methods in here might help though: http://api.rubyonrails.org/classes/ActiveRecord/Sanitization/ClassMethods.html